idseq_pipeline/commands/postprocess_functions.py

```diff
@@ -77,17 +79,32 @@ def accession2taxid(read_id, accession2taxid_dict, hit_type, lineage_map):
     species_taxid, genus_taxid, family_taxid = lineage_map.get(taxid, ("-100", "-200", "-300"))
     return species_taxid, genus_taxid, family_taxid
 
-def generate_taxid_fasta_from_accid(input_fasta_file, accession2taxid_path, lineagePath, output_fasta_file):
+def generate_taxid_fasta_from_accid(input_fasta_file, hit_summary_files, accession2taxid_path, lineagePath, output_fasta_file):
     accession2taxid_dict = shelve.open(accession2taxid_path)
     lineage_map = shelve.open(lineagePath)
+    nt_hit_summary_file = hit_summary_files[0]
+    nr_hit_summary_file = hit_summary_files[1]
```

Comment:
two long variable names that differ in only one letter -- generally error prone

usually we see this in
```
for my_wonderful_thing in my_wonderful_things:
   do_something(my_wonderful_thing)
```
where we are one typo away from calling `do_something` on the entire list

the shorter the variable name the less problematic this is --- for example `nt` and `nr` are ok since they differ by 50% :P)  though `nT` and `nR` would be better since they emphasize the difference

Reply:
ok